ironic-images/ipa-debian-bookworm/ipa-debian-bookworm.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# elements path - need to include both the ironic-python-agent-builder DIB packages
+# and our custom_elements packages
+# export ELEMENTS_PATH=/root/keekz-image-build/venv/share/ironic-python-agent-builder/dib:/root/keekz-image-build/ipa-build-20240801/custom_elements
+
+# distro version
+export DIB_RELEASE=bookworm
+
+# devuser element - https://github.com/openstack/diskimage-builder/tree/master/diskimage_builder/elements/devuser
+export DIB_DEV_USER_USERNAME=undercloud
+export DIB_DEV_USER_PASSWORD=secret_password
+export DIB_DEV_USER_PWDLESS_SUDO=yes
```

Comment:
```suggestion
```

I'd rather see this gone and utilizing https://docs.openstack.org/diskimage-builder/latest/elements/dynamic-login/README.html which is enabled by the IPA DIB by default. We can utilize `sshkey` and `rootpwd`.

Reply:
Switched to dynamic-login element 👍